arangodb-net-standard/Enums/HTTPCodes.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ArangoDBNetStandard.Enums
+{
+    public enum HTTPCodes : int
```

Comment:
Should this be introduced in it's own commit?

Also why not use the existing System.Net.HttpStatusCode?

Reply:
Initialially thought that ArnagoDB's use of Http codes was unique. But yeah, just checked, they are all the same. So will create a seperate issue to make this change to use HttpStatusCode instead of a custom enum.

Reply:
Oh wait. If the code returned in the model is the error code returned by ArangoDB, as documented here: https://www.arangodb.com/docs/stable/appendix-error-codes.html then I believe there is no point to change that property.
If it's the HTTP status code, then using System.Net.HttpStatusCode seems more appropriate indeed.
(We should really document that, we can possibly create a new issue.)

I strongly believe we should create a separate commit anyway to introduce any enum or changes representing those error codes. I don't see any reason to change them in that commit. It seems unrelated to the issue. Maybe missed something?

Reply:
So there is the Http Codes which are the same as ArnagoDB, and then there is the property "ErrorNum" which will be from the Appendix error codes. And that can be an enum aswell.

So two separate commits for those changes. I have already created these issues

Reply:
I checked, this code is the HTTP code. I notice you created a new issue for that. 👍